Test/PSNeo4j.Tests.ps1

```diff
@@ -187,16 +187,16 @@ Describe "Get-Neo4jLabel $PSVersion" {
 Describe "New-Neo4jNode $PSVersion" {
     Mock Invoke-Neo4jQuery -ModuleName PSNeo4j
     It 'Should create a new node' {
-        New-Neo4jNode -Label Service -InputObject @{
+        New-Neo4jNode -Label Service -PassThru -InputObject @{
             Name = 'Active Directory'
             Engineer = 'Warren Frame'
         }
         Assert-MockCalled Invoke-Neo4jQuery -ModuleName PSNeo4j -Exactly 1 -Scope It -ParameterFilter {
-            $Query -eq 'UNWIND {props} AS properties CREATE (node:Service) SET node = properties RETURN node'
-            $Parameters.keys -contains 'Name'
-            $Parameters.keys -contains 'Engineer'
-            $Parameters['Name'] -eq 'Active Directory'
-            $Parameters['Engineer'] -eq 'Warren Frame'
+            $Statements[0].Statement -eq 'UNWIND {props} AS properties CREATE (node:Service) SET node = properties RETURN node' -and
```

Comment:
@RamblingCookieMonster you might want to double check me on this bit

Reply:
Pester seems happy? : )  `-Statements $Statements` but will never have a singular `-Statement` so should be fine - thanks for catching all this nonsense : )